File: Data Structures & Algorithms/is-anagram/submission-1.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        int n1 = s.Length;
        int n2 = t.Length;
        if(n1!=n2)
        {
            return false;
        }
        int[] freq = new int[26];
        foreach(char ch in s)
        {
            freq[ch-'a']++;
        }
        foreach(char ch in t)
        {
            freq[ch-'a']--;
        }
        foreach(int i in freq)
        {
            if(i!=0)
            {
                return false;
            }
        }
        
        return true;
        

    }
}

File: Data Structures & Algorithms/kth-distinct-string-in-an-array/submission-1.py
class Solution:
    def kthDistinct(self, arr: List[str], k: int) -> str:
        cnt=0
        freq={}
        for word in arr:
            freq[word]=freq.get(word,0)+1
        for word in arr:
            if freq[word]==1:
                k-=1
                if k==0:
                    return word
        return ""


        